tests/test_interfaces.py

```diff
@@ -44,10 +44,10 @@ def get_grouped_edges(invitation, limit=1000, offset=0, **kwargs):
     return client
 
 def assert_float_arrays(array_A, array_B):
-    assert all([float(a) == float(b) for a, b in zip(array_A, array_B)])
+    assert all([float(a) == float(b) for a, b in zip(sorted(array_A), sorted(array_B))])
 
 def assert_str_arrays(array_A, array_B):
```

Comment:
assert arrays is already  implemented in another test: https://github.com/openreview/openreview-matcher/blob/master/tests/test_encoder.py#L30

Can you move this code to somewhere else it can be used by different tests?